### PR TITLE
[DOCS-7211] Collaboration Connector for Microsoft 365 2.0

### PR DESCRIPTION
**Pull request by Amy-Hyland**

### CODE REVIEW

**Review by dsibilio — DISMISSED** [review]

**Review by mpichura — DISMISSED** [review]
LGTM

File: microsoft-365/1.1/support/index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+title: Supported platforms
+---
+
+The following are the supported platforms for the Alfresco Collaboration Connector for Microsoft 365 version 1.1.3:
+
+| Version | Notes |
+| ------- | ----- |
+| Alfresco Content Services 7.2.0 | |
+| Alfresco Content Services 7.1.0 | |
+| Alfresco Content Services 7.0.0 | |
```
Comment (anxumalo) [inline microsoft-365/1.1/support/index.md:11]: According to the [Alfresco Product Support Status page](https://www.alfresco.com/services/subscription/technical-support/product-support-status), MS 365 1.1 is also compatible with ACS 7.3: 

```suggestion
| Alfresco Content Services 7.3.x | |
| Alfresco Content Services 7.2.x | |
| Alfresco Content Services 7.1.x | |
| Alfresco Content Services 7.0.x | |
```

File: microsoft-365/latest/support/index.md [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+
+The following are the supported platforms for the Alfresco Collaboration Connector for Microsoft 365 version 2.0.0:
+
+| Version | Notes |
+| ------- | ----- |
+| Alfresco Content Services 7.4.x | |
+| Alfresco Content Services 7.3.x | |
+| Alfresco Content Services 7.2.0 | |
+| Alfresco Content Services 7.1.0 | |
+| Alfresco Content Services 7.0.0 | |
```
Comment (anxumalo) [inline microsoft-365/latest/support/index.md:13]: Let's make these versions more generic to match the lines above:
```suggestion
| Alfresco Content Services 7.2.x | |
| Alfresco Content Services 7.1.x | |
| Alfresco Content Services 7.0.x | |
```

**Review by anxumalo — CHANGES_REQUESTED** [review]
Looks good - just a few minor fixes please

**Review by anxumalo — APPROVED** [review]

**Pull request merged**